### ISSUE TITLE
Custom Test Case 

### ISSUE THREAD

**Issue by DE0CH** [CP-STA/STAOJ#43]
Would be good if there's a page for people to run code against their custom test case, for those who can't be bothered to set up their own dev environment. 

![](https://cdn.discordapp.com/attachments/965976708186533930/1015567694977835109/Screenshot_2022-09-03_at_11.23.08.png)

Should be pretty straightforward to implement this in the frontend. @Daniel5055 What about the executioner?

**Comment by Daniel5055** [comment]
Yeah this would be fine to implement. Would just need to add an additional field to submissions like "customCases" or something. If it exists in the submission then executioner will read the testCases from there and write the output somewhere.

**Comment by Daniel5055** [comment]
Would each line be a separate test case or is the whole input a single test case?

**Comment by DE0CH** [comment]
I guess the whole input would be a single test case. It would work more like an online IDE where you can run code and get output. I am not really familiar with the use case cause I do my dev locally but @kay-ventris1 might have something to say about this.

**Comment by kay-liechtenstein** [comment]
> I guess the whole input would be a single test case.

Yes. In the example of CF there's `t` indicating the number of testcases given in a single input. However, majority of the problems I've written so far don't employ this style and one input contains one testcase. 

For example, if the problem is "output `n + 1` where `n` is given" and you have written code, you can only key only one number in the "Input:". You paste your code, you key `3` in "Input:", you click on "run", then `4` is shown in "Output:".
